### PR TITLE
HOTFIX: promote `agfs_scheme_12s` scope to superclass

### PR DESCRIPTION
**Pull request by jsugarman**
#### What
Enable agfs_scheme_12s (role) scope to return all applicable
scheme 12 fees

#### Ticket

[CBO-1479](https://dsdmoj.atlassian.net/browse/CBO-1479)

#### Why
hotfix for API interactive documentation (swagger docs)

### CODE REVIEW

File: app/models/fee/base_fee_type.rb [hunk 1]
```diff
@@ -32,6 +32,10 @@ class BaseFeeType < ApplicationRecord
     include CaseUpliftable
     include DefendantUpliftable
 
+    # TODO: update all agfs_scheme_10 fee types to also have agfs_scheme_12 roles
```
Comment (naseberry) [inline app/models/fee/base_fee_type.rb:35]: Is there a timeframe for completing this?

**Review by naseberry — APPROVED** [review]
👍

File: app/models/fee/base_fee_type.rb (hunk 1 above)
Comment (jsugarman) [reply app/models/fee/base_fee_type.rb:35]: am working on the "proper" fix now.

Comment (jsugarman) [reply app/models/fee/base_fee_type.rb:35]: i cherry-pick'ed this out of the [proper fix PR](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/3512)

**Pull request merged**